rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -143,8 +149,20 @@ def _load_OOV_params(self) -> None:
             if self.OOV_words:
                 self.OOV_words = [w.lower() for w in self.OOV_words]
 
+        # Additional vocabulary size to be kept reserved
+        self.additional_vocabulary_size = self.component_config[
+            "additional_vocabulary_size"
+        ]
+        if isinstance(self.additional_vocabulary_size, int):
```

Comment:
do we allow both `int` and `dict`?

Reply:
I suggest to always have `dict`

Reply:
Ahh good that you pointed it out, I had meant to remove this piece of code. Agreed, we should stick to `dict`.